users/models.py

```diff
@@ -62,6 +62,7 @@ class User(AbstractBaseUser):
     description = models.TextField(default='', blank=True)
     is_active = models.BooleanField(default=False)
     confirm_token = models.CharField(default=create_hash, max_length=40)
+    password_reset_token = models.CharField(default=create_hash, max_length=40)
```

Comment:
Ok, know we get an invalid 404. I hope it is better now. Sorry for Inconvenience.